### ISSUE TITLE
Screensharing on nerimity.com/app doesn't work for entire screen or most windows

### ISSUE THREAD

**Issue by christianlittle7** [Nerimity/nerimity-web#65]
When joining a voice call and selecting 'Share Screen', filling out options, and selecting 'Choose Window' works just fine as long you select a Chrome tab or Chrome Window.

If you select any other application (At least everything I've tried so far) or the entire screen, when you hit the 'Share' button it takes you back to the screen where you select your quality and Framerate with the 'Choose Window' Button.

Chrome Version: 123.0.6312.106
Apple M3 Pro running Sonoma 14.4

**Comment by christianlittle7** [comment]
Update! Looks like I figured it out! You have to manually go to to your system settings > Privacy > Screen & System Audio Recording and manually add chrome to the list of apps there.

Nerimity won't prompt, and doesn't give any errors about this though

**Comment by SupertigerDev** [comment]
In mac settings? Unfortunately, I don't have a mac

**Comment by christianlittle7** [comment]
Correct! It looks like Mac users would need to go into their settings to manually add chrome to be able to screen share. Nerimity doesn't prompt for that behavior.
